src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/DashboardController.cs

```diff
@@ -25,28 +23,28 @@ public IActionResult Index()
 
         public async Task<ViewResult> NewApplications()
         {
-            var userId = _contextAccessor.HttpContext.User.UserId();
+            var userId = HttpContext.User.UserId();
             userId = "temp"; //TODO: Can't access the user until staff idams is enabled
             var vm = await _orchestrator.GetNewApplicationsViewModel(userId);
             return View(vm);
         }
 
         public async Task<IActionResult> AssignToAssessor(Guid applicationId, int assessorNumber)
         {
-            var userId = _contextAccessor.HttpContext.User.UserId();
-            var userName = _contextAccessor.HttpContext.User.UserDisplayName();
+            var userId = HttpContext.User.UserId();
+            var userName = HttpContext.User.UserDisplayName();
 
             userId = "temp"; //TODO: Can't access the user until staff idams is enabled
             userName = "Joe Bloggs"; //TODO: Can't access the user until staff idams is enabled
 
             await _orchestrator.AssignApplicationToAssessor(applicationId, assessorNumber, userId, userName);
 
-            return RedirectToAction("ViewApplication", "Home", new { applicationId });
+            return RedirectToAction("ViewApplication", "Overview", new { applicationId });
         }
 
         public async Task<ViewResult> InProgressApplications()
         {
-            var userId = _contextAccessor.HttpContext.User.UserId();
+            var userId = HttpContext.User.UserId();
             userId = "temp"; //TODO: Can't access the user until staff idams is enabled
```

Comment:
Doesn't userId still resolve to something like 'unknown user' anyway? So this 'temp' line wouldn't be needed

Reply:
Awaiting iDAMS to work correctly. A ticket will be raised to sort this out.